src/renderer/components/+preferences/preferences-navigation/extensions-preferences-navigation-item.injectable.ts

```diff
@@ -3,10 +3,14 @@
  * Licensed under MIT License. See LICENSE in root directory for more information.
```

Comment:
Commit name explains technicality and not motivation.

Reply:
I would not change commit message because I find it a bit risky for commit history, but will keep this in mind for future reference.